### ISSUE TITLE
visibility:fieldname=visible does not work with metafields:order

### ISSUE THREAD

**Issue by malaire** [shannah/xataface#46]
Given following relationships.ini file, FieldA is not visible.

[SomeName]
SomeName.SomeID = "$SomeID"
metafields:order = FieldA
visibility:FieldA = visible

**Comment by shannah** [comment]
The metafields:order is taking precedent and hiding the field.

**Comment by malaire** [comment]
Exactly, which is a bug since I should be able to make that field visible if I need to.
